CHANGELOG.md

```diff
@@ -55,6 +67,8 @@
 
 ## **5.39.0** (2019-09-30)
 
+![Channel: stable](https://badgen.net/badge/channel/stable/green)
+
```

Comment:
Nope, can you tell me why?

Reply:
We put stable for the release patch because HUB part is fixed.